lib/sms_sender.rb

```diff
@@ -0,0 +1,47 @@
+class SmsSender
+  def initialize(recipient, text, login, password)
+    @recipient = recipient
+    @text = text
+    @login = login
+    @password = password
+  end
+
+  def send_sms
+    return unless Rails.env.production?
+    uri = URI.parse('http://svitsms.com/api/api.php')
+    phone = @recipient
+    text = @text
+
+    header = {
+      'Content-Type': 'text/xml'      
+    }
+    xml = <<~XML
+    <?xml version="1.0" encoding="utf-8"?>
+    <request>
+      <operation>SENDSMS</operation>
+      <message start_time="AUTO" end_time="AUTO" lifetime="4" rate="120" desc="My first campaign " source="SMS">
+        <body>#{text}</body> 
+        <recipient>#{phone}</recipient>
+      </message>
+    </request>
```

Comment:
Move it to constant

Reply:
what to constant? uri is the only one thing, i can move to a constant, everything else is dynamic

Reply:
XML. Check example [here](https://gist.github.com/galulex/ee6d057f27eff6375b9f#file-lesson1-rb-L3)

Reply:
everything is done